tests/src/test/scala/cats/tests/FoldableSuite.scala

```diff
@@ -140,6 +140,12 @@ abstract class FoldableSuite[F[_]: Foldable](name: String)(
       fa.toList should === (iterator(fa).toList)
     }
   }
+  
+  test(s"Foldable[$name] mkString_") {
+    forAll { (fa: F[Int]) =>
+      fa.mkString_("L[", ";", "]") should === fa.mkString("L[", ";", "]")
```

Comment:
now is the right hand side working? don't we need a toList or something?

Reply:
good catch. Since this is not my repo I can only edit files through github and my mental compiler is clearly not good enough.